services/ride-service/src/main/java/com/Harevich/ride_service/service/impl/PriceServiceIml.java

```diff
@@ -0,0 +1,65 @@
+package com.Harevich.ride_service.service.impl;
+
+import static com.Harevich.ride_service.util.constants.TariffMultiplyConstants.END_OF_HOUR_MULTIPLY_CONSTANT;
+import static com.Harevich.ride_service.util.constants.TariffMultiplyConstants.EVENING_MULTIPLY_CONSTANT;
+import static com.Harevich.ride_service.util.constants.TariffMultiplyConstants.MORNING_MULTIPLY_CONSTANT;
+import static com.Harevich.ride_service.util.constants.TimeTariffConstantValues.EVENING_PEAK_END;
+import static com.Harevich.ride_service.util.constants.TimeTariffConstantValues.EVENING_PEAK_START;
+import static com.Harevich.ride_service.util.constants.TimeTariffConstantValues.MINUTES_IN_HOUR_WITHOUT_INCREASED_TARIFF;
+import static com.Harevich.ride_service.util.constants.TimeTariffConstantValues.MORNING_PEAK_END;
+import static com.Harevich.ride_service.util.constants.TimeTariffConstantValues.MORNING_PEAK_START;
+import static com.Harevich.ride_service.util.constants.TimeTariffConstantValues.TARIFF_PER_KM;
+
+import com.Harevich.ride_service.service.GeolocationService;
+import com.Harevich.ride_service.service.PriceService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+
+@Service
+@RequiredArgsConstructor
+public class PriceServiceIml implements PriceService {
+
+    private final GeolocationService geolocationService;
+
+    @Override
+    public BigDecimal getPriceByTwoAddresses(String from, String to, LocalDateTime currentTime) {
+        double distance = geolocationService.getRouteDistanceByTwoAddresses(from,to);
+        BigDecimal price = BigDecimal.valueOf(distance*TARIFF_PER_KM);
+        //не знаю, вроде криво, но иначе будет еще более криво
+        BigDecimal finalPrice =
+                increaseTariffByMorningPeak(currentTime,
+                    increaseTariffByEveningPeak(currentTime,
+                        increaseTariffByTheEndOfHour(currentTime,price)));
+        return finalPrice;
+    }
```

Comment:
Не стоит забывать об автоформатировании. Комментарий следует убрать.
```suggestion
    public BigDecimal getPriceByTwoAddresses(String from, String to, LocalDateTime currentTime) {
        double distance = geolocationService.getRouteDistanceByTwoAddresses(from, to);
        BigDecimal price = BigDecimal.valueOf(distance * TARIFF_PER_KM);
        //не знаю, вроде криво, но иначе будет еще более криво
        BigDecimal finalPrice =
                increaseTariffByMorningPeak(currentTime,
                    increaseTariffByEveningPeak(currentTime,
                        increaseTariffByTheEndOfHour(currentTime, price)));
        return finalPrice;
    }
```